### ISSUE TITLE
Big_Buck_Bunny_1080_10s_1MB.mp4 missing

### ISSUE THREAD

**Issue by jasonsansone** [HaveAGitGat/Tdarr#327]
**Describe the bug**

`Error: ENOENT: no such file or directory, open '/opt/assets/app/testfiles/Big_Buck_Bunny_1080_10s_1MB.mp4`

On launching Tdarr_Server, the binary looks for Big_Buck_Bunny_1080_10s_1MB.mp4.  It is not downloaded and installed by Tdarr_Updater.  There is no folder at `/opt/assets/app/testfiles/`.

**To Reproduce**
Install v2 on linux.  Run Tdarr_Server.

OS: Ubuntu 20.04.1 LTS as LXC container on Proxmox 5.4.78-2
Tdarr Version: 2.00.00

**Comment by wsegatto** [comment]
There you go. I have attached the file.

https://user-images.githubusercontent.com/35738729/105752470-17f10700-5f26-11eb-9f78-52e99838f3f6.mp4

Maybe a tip to help here, because it worked for me and I'm using Debian and it worked:

a) Delete everything but the updater.
b) In the updater folder e.g. /opt/Tdarr/Tdarr_Updater, run it.
c) Don't run the server/node/webgui from this folder. Instead, navigate into them and just run e.g. ./Tdarr_Server. This will make the folders like assets, logs, etc. to be created in /opt/Tdarr, not in /opt which seems your case.

Let me know if this helps.

**Comment by jasonsansone** [comment]
> There you go. I have attached the file.
> 
>  Big_Buck_Bunny_1080_10s_1MB.mp4 

Thank you.  My concern is that the code throws an error for an asset it didn't download.  I believe that either the installer should download the file, or the made optional and the module fail more gracefully when the mp4 file is not present.

> Maybe a tip to help here, because it worked for me and I'm using Debian and it worked:
> 
> a) Delete everything but the updater.
> b) In the updater folder e.g. /opt/Tdarr/Tdarr_Updater, run it.
> c) Don't run the server/node/webgui from this folder. Instead, navigate into them and just run e.g. ./Tdarr_Server. This will make the folders like assets, logs, etc. to be created in /opt/Tdarr, not in /opt which seems your case.
> 
> Let me know if this helps.

Yes, that works.  However, I am surprised that the module binaries need to be executed from such specific locations.  I would advise revising the code to utilize more relative paths so they can be executed from any location.

**Comment by wsegatto** [comment]
Glad to help. Yes, I have mentioned that in #317, but I'm gathering data on other issues before coming back to this one. So far the workaround would be to "cd" first and then run it. Not so hard to overcome, but I don't think that's the intended behavior. I also got files in /opt that I didn't want. Plus, /opt is owned by root while /opt/Tdar by pi.

**Comment by HaveAGitGat** [comment]
The folder structure bug when running modules from a different dir should be fixed in 2.00.01. I've made some changes to Tdarr_Updater - the new version allows downloading of a new package without it being 'latest'.  So if you download the new updater (http://tdarr.io/docs/), and specify 2.00.01 in the config, it will grab that package. If you leave it as 'latest' it will still grab 2.00.00. Once fixes have been verified, I can set 2.00.01 as 'latest'.

```
{
  "platform_arch": "default",
  "modules": {
    "Tdarr_Desktop": {
      "enabled": true,
      "requiredVersion": "2.00.01",
      "currentVersion": "2.00.01"
    },
    "Tdarr_Node": {
      "enabled": true,
      "requiredVersion": "2.00.01",
      "currentVersion": ""
    },
    "Tdarr_Server": {
      "enabled": true,
      "requiredVersion": "2.00.01",
      "currentVersion": ""
    },
    "Tdarr_WebUI": {
      "enabled": true,
      "requiredVersion": "2.00.01",
      "currentVersion": ""
    }
  }
}
```

**Comment by jasonsansone** [comment]
Fixed in 2.00.01
